climpred/stats.py

```diff
@@ -155,71 +155,63 @@ def xr_rm_poly(ds, order, dim='time'):
     Returns:
         xarray object with polynomial fit removed.
     """
-    _check_xarray(ds)
+    _check_xarray(ds)  # this could be a decorator I think?
```

Comment:
That's a good point. I need to read up on decorators. Haven't really used them yet.